### ISSUE TITLE
app is unstable after updating to 8.2.4

### ISSUE THREAD

**Issue by rrrafalsz** [mapsme/omim#8671]
[bug report]
i just updated from 8.1.3 to 8.2.4. before, the app was working relatively fine (but slower than previous versions). after the update, the app is unusable. 

version: 8.2.4 google
phone:
HUAWEI LUA-L21
android 5.1

issue in detail:

* trying to update my maps by going to menu > download maps causes the app to freeze and stop responding
* going to bookmark lists causes the app to freeze and stop responding
* pressing the discovery button causes the app to freeze and stop responding
* opening search causes the app to freeze and stop responding
* bookmarks take several seconds to load after the launch of the app

basically, as long as i am navigating around the map, it works fine. after i press any menu button, it crashes.

how do i downgrade?

PS. also, wtf, you introduced ads below place descriptions and won't even mention it in the changelog?

**Comment by thany** [comment]
As for your wtf: yep, they really do give the finger to people who have paid for the app back when it was a paid-for app called MapsWithMe Pro. 👉🤨👈

**Comment by ForkWithMe** [comment]
The issue is outdated. I kindly suggest to update the app. If you really want to help, please attach logs.
